webapi/src/main/twirl/org/knora/webapi/messages/twirl/queries/sparql/v2/changeResourceMetadata.scala.txt

```diff
@@ -90,7 +90,7 @@ WHERE {
 
         @maybeLastModificationDate match {
             case Some(lastModificationDate) => {
-                ?resource knora-base:lastModificationDate "@lastModificationDate"^^xsd:dateTime .
+                ?resource knora-base:lastModificationDate ?lastModificationDate .
```

Comment:
Is the last modification date still typed in the triplestore with this change?

Reply:
yes, I'm just using whatever is in the triplestore already rather than the value passed to he template, when deleting the old last modification date. (The idea is to make this independent of the serialization in the triplestore)

Reply:
Ah, I misread it. It's only inside the DELETE and WHERE clause, the INSERT still has the typed version! Ignore my comment then.